tests/integration/test_users.py

```diff
@@ -7,6 +7,21 @@
 from github3.exceptions import MethodNotAllowed
 
 
+class TestGPGKey(IntegrationHelper):
+
+    """Integration tests for methods of the GPGKey class."""
+
+    def test_delete(self):
+        """Test the ability to delete a GPG key."""
+        self.token_login()
+        cassette_name = self.cassette_name('delete')
+        with self.recorder.use_cassette(cassette_name):
+            gpg_key = self.gh.gpg_key(407238)
```

Comment:
And this test can look a lot like the `create_gpg_key` test. It's good to have it anyway.